cli/src/main/kotlin/commands/UploadResultToSw360Command.kt

```diff
@@ -85,9 +101,32 @@ class UploadResultToSw360Command : CliktCommand(
         getProjectWithPackages(ortResult).forEach { (project, pkgList) ->
             val linkedReleases = pkgList.mapNotNull { pkg ->
                 val name = createReleaseName(pkg.id)
-                sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
+                val release = sw360ReleaseClient.getSparseReleaseByNameAndVersion(name, pkg.id.version)
                     .flatMap { sw360ReleaseClient.enrichSparseRelease(it) }
                     .orElseGet { createSw360Release(pkg, sw360ReleaseClient) }
+
+                if (attachSources) {
+                    val tempDirectory = createOrtTempDir(pkg.id.toPath())
+                    try {
+                        val sourceCodeArchive = saveSourcesTemporarily(pkg, tempDirectory)
+
+                        if (sourceCodeArchive != null) {
+                            uploadAttachmentToRelease(
```

Comment:
Please make this simply return a `Boolean` instead of a `SW360Release` and handle the logging in the caller.

Reply:
I completely removed the function and put it to the caller. Now I handle the logging based on the `uploadResult` in the caller directly.